homeassistant/components/unifiprotect/strings.json

```diff
@@ -67,7 +67,7 @@
         "step": {
           "start": {
             "title": "UniFi Protect Early Access enabled",
-            "description": "You are either running an Early Access version of UniFi Protect (v{version}) or opt-ed into a release channel that is not the Official Release Channel. [Home Assistant does not support Early Access versions](https://www.home-assistant.io/integrations/unifiprotect#about-unifi-early-access), so you should immediately switch to the Official Release Channel. Accidentally upgrading to an Early Access version can break your UniFi Protect integration.\n\nBy submitting this form, you have switched back to the Official Release Channel or agree to run an unsupported version of UniFi Protect, which may break your Home Assistant integration at any time."
+            "description": "You are either running an Early Access version of UniFi Protect (v{version}) or opt-ed into a release channel that is not the Official Release Channel.\n\nAs these Early Access may not be tested yet, upgrading to it may cause the UniFi Protect integration to behave unexpectedly. [Read more about Early Access and Home Assistant](https://www.home-assistant.io/integrations/unifiprotect#about-unifi-early-access).\n\nSubmit to dismiss this message."
```

Comment:
```suggestion
            "description": "You are either running an Early Access version of UniFi Protect (v{version}) or opt-ed into a release channel that is not the Official Release Channel.\n\nAs these Early Access may not be tested yet, using it may cause the UniFi Protect integration to behave unexpectedly. [Read more about Early Access and Home Assistant](https://www.home-assistant.io/integrations/unifiprotect#about-unifi-early-access).\n\nSubmit to dismiss this message."
```